android/java/res/layout/brave_rewards_onboarding_modal.xml

```diff
@@ -64,36 +62,25 @@
 	            android:text="@string/take_quick_tour"
 	            android:textSize="14sp"
 	            android:textStyle="bold"
-	            android:textColor="@color/brave_rewards_modal_theme_color"/>
+	            android:fontFamily="@font/poppins_regular"
+	            android:textColor="@color/rewards_panel_action_color"/>
 
-	        <android.widget.Button
+	        <TextView
```

Comment:
not so important, but since this control is not button, but text view now, maybe it worth to rename  `btn_brave_rewards` => `brave_rewards_text` to avoid confusing?

or that `TextView` in fact is used like a button with main purpose to be pressed? - then ok to keep it as `btn_brave_rewards`

Reply:
looking on more files, I see it is used as a button, resolving